cairo_vm_hints/src/hints/lib/verifiers/evm/account_verifier.rs

```diff
@@ -0,0 +1,230 @@
+use cairo_vm::{
+    hint_processor::builtin_hint_processor::{
+        builtin_hint_processor_definition::HintProcessorData,
+        hint_utils::{get_integer_from_var_name, get_ptr_from_var_name, insert_value_from_var_name, insert_value_into_ap},
+    },
+    types::{exec_scope::ExecutionScopes, relocatable::MaybeRelocatable},
+    vm::{errors::hint_errors::HintError, vm_core::VirtualMachine},
+    Felt252,
+};
+use num_bigint::BigUint;
+use num_traits::Num;
+use std::collections::HashMap;
+
+use crate::{
+    hint_processor::models::proofs::Proofs,
+    hints::{
+        lib::utils::{count_leading_zero_nibbles_from_hex, split_128},
+        vars,
+    },
+};
+
+pub const HINT_BATCH_ACCOUNTS_LEN: &str = "len(batch[\"accounts\"])";
+
+pub fn hint_batch_accounts_len(
+    vm: &mut VirtualMachine,
+    exec_scopes: &mut ExecutionScopes,
+    _hint_data: &HintProcessorData,
+    _constants: &HashMap<String, Felt252>,
+) -> Result<(), HintError> {
+    let batch = exec_scopes.get::<Proofs>("batch")?;
+
+    insert_value_into_ap(vm, Felt252::from(batch.accounts.len()))
+}
+
+pub const HINT_GET_ACCOUNT_ADDRESS: &str =
+    "account = batch.accounts[ids.idx]\nsegments.write_arg(ids.address, [int(x, 16) for x in account.address])";
+
+pub fn hint_get_account_address(
+    vm: &mut VirtualMachine,
+    exec_scopes: &mut ExecutionScopes,
+    hint_data: &HintProcessorData,
+    _constants: &HashMap<String, Felt252>,
+) -> Result<(), HintError> {
+    let batch = exec_scopes.get::<Proofs>("batch")?;
+    let idx: usize = get_integer_from_var_name(vars::ids::IDX, vm, &hint_data.ids_data, &hint_data.ap_tracking)?
+        .try_into()
+        .unwrap();
+    let account = batch.accounts[idx].clone();
+    let address = BigUint::from_str_radix(&account.address.to_string(), 16).unwrap();
```

Comment:
why we convert to string and later convert back to address?
address = account.address

Reply:
Can we do `vm.write_arg(address_ptr, &account.address)?` directly?

Reply:
```
account = batch.accounts[ids.idx]
```
u didnt do exec_scopes.write in this hint if u have this in exec scope the issue below with idx is resolving automatically